lib/mtl/rails/view_helpers.rb

```diff
@@ -226,17 +226,21 @@ def mtl_button(label, url, options = {})
       # @param menu [String, Boolean] (HTML ID) references the id of the aside menu / default
       #   nav menu to show / hide on mobile devices. The default used is `nav-menu`.
       #   When set to `false`, this button is skipped.
+      # @param class [String, Array] (HTML CLASS) additional, custom css class on header
       # @yield Additional content to be rendered as part of the header
       # @return [String] HTML safe string
-      def mtl_header(title = translate('.title', default: 'Menu'),
-                     back: false, menu: 'nav-menu', &block)
+      def mtl_header(title = translate('.title', default: 'Menu'), **options, &block)
```

Comment:
we can still make backwards incompatible changes now, I'm open to change this to `mtl_header(options = {})` and that `:title` is simply yet another option?